spec/config/lint-on-fly-spec.coffee

```diff
@@ -0,0 +1,11 @@
+describe "The Lint on the Fly Configuration Option", ->
```

Comment:
I honestly hadn't thought about it. Would you prefer config tests in one file, because I kinda like the smaller files.